pkg/platform/packet/template.go

```diff
@@ -125,8 +135,27 @@ module "worker-{{ $pool.Name }}" {
 
   ssh_keys  = {{$.SSHPublicKeys}}
 
+  {{- if $pool.CLCSnippets}}
+  clc_snippets = [
+	{{- range $clc_snippet := $pool.CLCSnippets}}
+    <<EOF
+{{ $clc_snippet }}
+EOF
+    ,
+  {{- end}}
+  ]
+  {{- end }}
+
   cluster_name = "{{$.Config.ClusterName}}"
-  tags         = {{$.Tags}}
+
+  {{- if $pool.Tags }}
+  tags = [
```

Comment:
It seems defining tags makes the `lokoctl-version` tag disappear. I think we need to append the lokoctl-versin tag to the worker pool tags too. We should fix this.

Reply:
done.

Reply:
https://github.com/kinvolk/lokomotive/blob/imran/feature-parity-packet/pkg/platform/packet/packet.go#L231-#L234

Reply:
We only append version tag to controller tags. Tags for worker pools have never been there I think.